lib/rust/profiler/data/src/bin/intervals.rs

```diff
@@ -0,0 +1,101 @@
+//! Tool that generates interval reports from profiling data.
+//!
+//! # Usage
+//!
+//! The tool reads a JSON-formatted event log from stdin, and writes a report to stdout.
+//!
+//! For example:
+//!
+//! ```console
+//! ~/git/enso/data $ cargo run --bin intervals < profile.json | less
+//! ```
```

Comment:
I have here the same comment as to the `devtools.rs` file – I'd be very thankful if you describe the context more, so people that are new to the codebase have easy time understanding what are all the components here, including what `profile.json` is and how to get it (this can, ofc, be a redirection to another docs).